public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    },
+    {
+      "src": "logo192.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
+}
```

Comment:
- [ ] As highlighted by the previous reviewers, one of the requirements is to delete all files that were created with the create react command that won't be relevant to this project. In light of this, kindly delete the following files in the _**public**_ directory:

- In the `public` directory:
  - favicon.ico
  - logo192.png
  - logo512.png
  - manifest.json
  - robots.txt